pystiche_papers/li_wand_2016/_loss.py

```diff
@@ -105,6 +124,36 @@ def style_loss(
     target_transforms: Optional[Iterable[transforms.Transform]] = None,
     score_weight: Optional[float] = None,
 ) -> ops.MultiLayerEncodingOperator:
+    r"""Style_loss from :cite:`LW2016`.
+
+    Args:
+        impl_params: If ``True``, uses the parameters used in the reference
+            implementation of the original authors rather than what is described in
+            the paper. For details see below.
+        multi_layer_encoder: Pretrained :class:`~pystiche.enc.MultiLayerEncoder`. If
+            omitted,  the default
+            :func:`~pystiche_papers.li_wand_2016.multi_layer_encoder` is used. Defaults
+            to ``None``.
+        layers: Layers from which the encodings of the ``multi_layer_encoder`` should be
+            taken. If omitted, the defaults is used. Defaults to
+            ``("relu3_1", "relu4_1")``.
+        layer_weights: Layer weights of the operator. Defaults to ``"sum"``.
+        patch_size: Size of the patch.
+        stride: Stride of the convolution. If omitted, the stride is determined with
+            respect to `impl_params``. For details see below.
+        target_transforms: Optional augemntation transformations for the target. If
+            omitted, the transfomrms are determined with respect to `impl_params``. For
+            details see below.
+        score_weight: Score weight of the operator. If omitted, the score_weight is
+            determined with respect to `impl_params``. For details see below.
+
+    If ``impl_params is True`` , a stride of 2 is used instead of 1, a score_weight of
+    1e-4 instead of the 1e0, and an additional score correction factor of 1.0 / 2.0 is
+    used instead of none. Additionally, no transformations are used in the scaling and
+    rotation. In the paper two additional transformations are used in the scaling and
+    the rotation. Furthermore, normalized patches are used instead of the unnormalized
```

Comment:
You need to link to `extract_normalized_patches` or no one will understand what "normalized patches" are.